frontend/ui/elements/filemanager_menu_order.lua

```diff
@@ -56,6 +56,7 @@ local order = {
         "version",
         "help",
         "----------------------------",
+        "system_statistics",
```

Comment:
Incidentally, does system_statistics go with exit? I'm inclined to think it's either its own thing or part of version and help. Then again, some programs (Opera/Blink) group help and about with exit.